lua/plenary/path.lua

```diff
@@ -115,7 +124,7 @@ local function _normalize_path(filename, cwd)
     until idx > #parts
 
     local prefix = ""
-    if is_absolute(filename, path.sep) or #_split_by_separator(cwd) == initial_up_count then
+    if is_absolute or #split_without_disk_name(cwd) == initial_up_count then
```

Comment:
this `is_absolute` looks wrong should still be `is_absolute(filename, path.sep)` if you ask me

I think the other stuff is good. Thanks :)

Reply:
This `is_absolute` is a variable from line 102 where it's initialized like this
```
local is_absolute = is_absolute(filename, path.sep)
```
That's my first lua experience, so I might be wrong, I thought that it's gonna be the same values anyway `is_absolute` and `is_absolute(filename, path.sep)`. So why not use already calculated value?
Please correct me if I'm wrong.

Reply:
Oh i overlooked that. Yeah can you rename that var? You are shadowing here and that is super confusing. just name it `is_abs` or so. 

You are btw also shadowing `filename` here `local split_without_disk_name = function(filename)` can you also do a rename there. Thanks :)